src/Utils/filterByLatest.js

```diff
@@ -1,13 +1,14 @@
+import { formatDate } from "./formatDate";
+
 export const filterByLatest = (filteredArray, byLatest) => {
   const withouFiltered = [...filteredArray];
-  if (byLatest === "") {
-    return withouFiltered;
-  } else {
+  if (byLatest !== "") {
     withouFiltered.sort((a, b) => {
-      let date1 = a.snippet.publishedAt.slice(0, 10).split("-").join("");
-      let date2 = b.snippet.publishedAt.slice(0, 10).split("-").join("");
+      let date1 = formatDate(a.snippet.publishedAt);
+      let date2 = formatDate(b.snippet.publishedAt);
```

Comment:
Use const if values are not re initialized

Reply:
Noted.